src/test/java/seedu/multitasky/logic/commands/EditCommandTest.java

```diff
@@ -31,80 +36,68 @@
 
     private Model model = new ModelManager(new TypicalEntries().getTypicalEntryBook(), new UserPrefs());
 
-    /*
     @Test
     public void execute_allFieldsSpecifiedUnfilteredList_success() throws Exception {
         Entry editedEntry = new EntryBuilder().build();
```

Comment:
hmm not sure why eclipse didn't catch these, but the call should be `Entry editedEntry = EntryBuilder.build();` instead.